File: Python/String-1/first_half.py
"""


Given a string of even length, return the first half. So the string "WooHoo" yields "Woo".


first_half('WooHoo') → 'Woo'
first_half('HelloThere') → 'Hello'
first_half('abcdef') → 'abc'

@author unobatbayar
"""
def first_half(str):
  half = len(str)/2
  return str[:half]

File: CodingBat/Java/String-1/firstHalf.java
/**
 
Given a string of even length, return the first half. So the string "WooHoo" yields "Woo".


firstHalf("WooHoo") → "Woo"
firstHalf("HelloThere") → "Hello"
firstHalf("abcdef") → "abc"

 */

public String firstHalf(String str) {
    int length = str.length()/2;
    return str.substring(0, length);
  }
  